test/resources/refund.test.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable func-names */
+import { expect } from 'chai';
+import * as setupPolly from '../setup_polly';
+import EasyPost from '../../src/easypost';
+import Fixture from '../helpers/fixture';
+import NotImplementedError from '../../src/errors/not_implemented';
+
+describe('Refund Resource', function () {
+  setupPolly.startPolly();
+
+  before(function () {
+    this.easypost = new EasyPost(process.env.EASYPOST_TEST_API_KEY);
+  });
+
+  beforeEach(function () {
+    const { server } = this.polly;
+    setupPolly.stripCreds(server);
+  });
+
+  it('create a refund', async function () {
+    const shipment = await new this.easypost.Shipment(Fixture.oneCallBuyShipment()).save();
+
+    const retrievedShipment = await this.easypost.Shipment.retrieve(shipment.id);
+
+    const refundData = {
+      carrier: Fixture.usps(),
+      tracking_codes: retrievedShipment.tracking_code,
+    };
+
+    const refunds = await this.easypost.Refund.save(refundData);
+
+    expect(refunds).to.be.an.instanceOf(Array);
```

Comment:
Let's not assert this, no need (really it's unfortunate that this endpoint returns an array).

Reply:
I can remove it, but other libs have this assertion.

Reply:
Eh, sure - either way